docs/auth/atlassian/provider.md

```diff
@@ -47,6 +47,11 @@ auth:
         clientId: ${AUTH_ATLASSIAN_CLIENT_ID}
         clientSecret: ${AUTH_ATLASSIAN_CLIENT_SECRET}
         scope: ${AUTH_ATLASSIAN_SCOPES}
+        signIn:
+          resolvers:
+            - resolver: emailMatchingUserEntityProfileEmail
+            - resolver: emailLocalPartMatchingUserEntityName
+            - resolver: usernameMatchingUserEntityName
```

Comment:
I'm feeling there's a risk that some people might copy this verbatim, which is never what you'd want. Do you agree and any ideas for a different way to present these?

Reply:
They will for sure copy this as is, I've seen that happen with the migration guide already when auth issues come up on Discord. I didn't realize that having them like this was problematic?

Reply:
I've updated each with this comment: `# typically you would pick one of these`